app/screens/post_options/post_options.tsx

```diff
@@ -97,12 +97,16 @@ const PostOptions = ({
         );
     };
 
+    // This fixes opening "post options modal" on top of "thread modal"
+    const additionalSnapPoints = location === Screens.THREAD ? 3 : 2;
```

Comment:
does it apply to Android & iOS?

Reply:
Yes, I have tested it on both iOS and Android.